cmake/HWMALLOCConfig.cmake.in

```diff
@@ -4,5 +4,4 @@ list(APPEND CMAKE_MODULE_PATH ${HWMALLOC_MODULE_PATH})
 include(CMakeFindDependencyMacro)
 set(NUMA_LIBRARY     @NUMA_LIBRARIES@)
 set(NUMA_INCLUDE_DIR @NUMA_INCLUDE_DIRS@)
-find_dependency(NUMA)
```

Comment:
hmm, do we not need this?

Reply:
The build on Mac will fail if I leave it there. Strange, because it is not marked `REQUIRED`. I wonder why that find is placed after using the variables `NUMA_LIBRARIES`/`NUMA_INCLUDE_DIRS`.
I can try to build on PizDaint to see if I can remove that line.

Reply:
I can try to build it locally, as well